test/org/sagebionetworks/bridge/models/accounts/ParticipantOptionsLookupTest.java

```diff
@@ -0,0 +1,149 @@
+import org.sagebionetworks.bridge.dao.ParticipantOption.SharingScope;
+
+import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
+
+public class ParticipantOptionsLookupTest {
+    
+    @Test
+    public void getDefaultValues() {
+        ParticipantOptionsLookup lookup = new ParticipantOptionsLookup(Maps.newHashMap());
+        
+        SharingScope scope = lookup.getEnum(SHARING_SCOPE, SharingScope.class);
+        assertEquals(SharingScope.NO_SHARING, scope);
+        
+        Boolean email = lookup.getBoolean(EMAIL_NOTIFICATIONS);
+        assertEquals(Boolean.TRUE, email);
+        
+        String identifier = lookup.getString(EXTERNAL_IDENTIFIER);
+        assertNull(identifier);
+        
+        Set<String> dataGroups = lookup.getStringSet(DATA_GROUPS);
+        assertTrue(dataGroups.isEmpty());
+        
+        String lang = lookup.getString(LANGUAGES);
+        assertNull(lang);
+    }
+    
+    @Test
+    public void getValuesWithWrongAccessor() {
+        ParticipantOptionsLookup lookup = new ParticipantOptionsLookup(Maps.newHashMap());
+        
+        ParticipantOption e = lookup.getEnum(SHARING_SCOPE, ParticipantOption.class);
+        assertNull(e);
+
+        Boolean b = lookup.getBoolean(EXTERNAL_IDENTIFIER);
+        assertEquals(Boolean.FALSE, b);
+        
+        // This happens because we store everything as a string, so we're going to get
+        // back a string here based on the enumeration name.
+        String s = lookup.getString(SHARING_SCOPE);
+        assertEquals("NO_SHARING", s);
+        
+        String s2 = lookup.getString(DATA_GROUPS);
+        assertNull(s2);
+        
+        Set<String> set = lookup.getStringSet(EXTERNAL_IDENTIFIER);
+        assertTrue(set.isEmpty());
+        
+        LinkedHashSet<String> set2 = lookup.getOrderedStringSet(EXTERNAL_IDENTIFIER);
+        assertTrue(set2.isEmpty());
+    }
+    
+    @Test
+    public void getStringWithValue() {
+        ParticipantOptionsLookup lookup = setupLookup(EXTERNAL_IDENTIFIER, "foo");
+        
+        assertEquals("foo", lookup.getString(EXTERNAL_IDENTIFIER));
+    }
+
+    @Test
+    public void getStringSetWithValue() {
+        ParticipantOptionsLookup lookup = setupLookup(DATA_GROUPS, "foo1,foo2,foo3");
+        
+        LinkedHashSet<String> set = TestUtils.newLinkedHashSet("foo1","foo2","foo3");
+        assertEquals(set, lookup.getStringSet(DATA_GROUPS));
+    }
+    
+    @Test
+    public void getOrderedStringSetWithValue() {
+        ParticipantOptionsLookup lookup = setupLookup(LANGUAGES, "en,fr,es");
+        
+        LinkedHashSet<String> set = TestUtils.newLinkedHashSet("en","fr","es");
+        assertEquals(set, lookup.getOrderedStringSet(LANGUAGES));
+    }
+
+    @Test
+    public void getBooleanWithValue() {
+        ParticipantOptionsLookup lookup = setupLookup(EMAIL_NOTIFICATIONS, "false");
+        
+        assertEquals(Boolean.FALSE, lookup.getBoolean(EMAIL_NOTIFICATIONS));
+    }
```

Comment:
Boolean.valueOf() is asymmetrical https://docs.oracle.com/javase/7/docs/api/java/lang/Boolean.html#valueOf%28java.lang.String%29 I'd recommend testing both true and false in unit tests.